package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ad4m-host",
-  "version": "0.0.22",
+  "version": "0.0.24",
```

Comment:
```suggestion
  "version": "0.0.23",
```

Reply:
On the github releases, 0.0.23 has already been published but I guess the version number was not incremented